packages/x-components/src/x-modules/related-tags/components/related-tags.vue

```diff
@@ -60,7 +65,7 @@
     components: { RelatedTag },
     mixins: [xComponentMixin(relatedTagsXModule)]
   })
-  export default class RelatedTags extends Vue {
+  export default class RelatedTags extends Mixins(dynamicPropsMixin(['relatedTagClass'])) {
```

Comment:
What do you think about renaming the `relatedTagClass` to be more generic?
`itemClass` or something like that, that way, if we add the prop to other list components all will have a similar API.

Reply:
Good point! I've renamed it to `itemClass`.